### PR TITLE
Add InnerSource SIG Docusaurus Skeleton Build

### PR DESCRIPTION
**Pull request by mcleo-d**
## Description
This pull request updates `docusaurus v1` to `docusaurus v2` using the [FINOS Project Blueprint](https://github.com/finos/software-project-blueprint) as a baseline template.

## Additional Changes
As part of this pull request ...

- Markdown file metadata has been added to each page to describe `title` and page `id`
- `resources.md` has been renamed to `awesome.md`
- In `docusaurus.config.js`, `projectName` now equals `InnerSource SIG` and `projectSlug` now equals `innersource-sig`

## Issues Found
In `project-maturity-matrix.md` lines 80 to 82, hyperlinks to the following files were missing, which means `https://` was added to overcome a MD parsing error.

```
- Examples:
  - [Project Lifecycle Definition](https://)
  - [Project Activation](https://)
```

### CODE REVIEW

File: website/.docusaurus/client-modules.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export default [
```
Comment (amaltson) [inline website/.docusaurus/client-modules.js:1]: Does this `.docusaurus` directory need to committed? Or is this generated code that we could add to the `.gitignore`?

File: website/sidebars.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
```
Comment (amaltson) [inline website/sidebars.js:1]: For generating the actual website and HTML to make it a GitHub site, do you prefer having it on the main branch or should we maybe consider having this go to the separate `gh-pages` branch and have a GitHub Action to perform a deployment?

**Review by amaltson — CHANGES_REQUESTED** [review]
This is really great, thank you for jumping in and getting this going @mcleo-d. I do see a fair amount of changes in one PR, maybe consider breaking this up into a couple of smaller PRs so it's easier to review and merge, maybe something like:

1. PR for the formatting updates, I assume you ran Prettier.
2. PR to add the initial Docsaurus dependency and configuration file.
3. PR to generate the Docusaurus website itself.

**Comment by amaltson** [comment]
Hey @mcleo-d, what's the standard deployment mechanism for FINOS Docusaurus projects again? Do you use GitHub Pages or was it Netlify? Thanks!

**Comment by amaltson** [comment]
Whoops, sorry I missed that!

**Comment by godds** [comment]
See #60

**Pull request merged**